test/export-api-def.test.js

```diff
@@ -102,6 +104,25 @@ describe('loopback:export-api-def generator', function() {
     });
   });
 
+  describe('running on project with onging async operations', function() {
+    var PROCESS_EXIT_STUB;
+
+    before(function stubProcessExit() {
```

Comment:
I think you need to stub `process.exit` for all tests (i.e. in the top-level `describe` block in this file). Otherwise the earlier tests exit the process before the test suit is over.

You can verify this fact by reviewing [CI build log](https://travis-ci.org/strongloop/generator-loopback/jobs/269885685) and searching for "ongoing" - the test output does not contain this word, which means the test was not run at all.